packages/test-framework/src/react-testing-library-support/mocks/checkout.mock.ts

```diff
@@ -340,7 +340,7 @@ const checkoutWithMultiShippingCart = {
         },
     },
     customer,
-    consignment: [],
+    consignments: [],
```

Comment:
`consignment` does not change the test results because `consignments` is still `undefined`.
Update to reflect the correct interface.